DependencyInjection/HttplugExtension.php

```diff
@@ -45,10 +57,22 @@ public function load(array $configs, ContainerBuilder $container)
             }
 
             $def = $container->register('httplug.client.'.$name, DummyClient::class);
-            $def->setFactory([new Reference($arguments['factory']), 'createClient'])
-                ->addArgument($arguments['config']);
+
+            if (empty($arguments['plugins'])) {
```

Comment:
Do we want this? An alternative could be to send everything to PluginClientFactory so every client is an instance of PluginClient.